src/libs/ajax/leonardo/models/app-models.ts

```diff
@@ -25,6 +25,34 @@ export type AppStatus =
   | 'STOPPED'
   | 'STARTING';
 
+export type DisplayAppStatus =
```

Comment:
Should we change this to AppStatusDisplay?
Seems like we could be slightly more consistent with the naming convention.

Either that, or all of our "display" types should start with display... But I'm not sure if I like that idea.

Reply:
Okay this may help - in "DisplayAppStatus"
In this, Display is a verb (hinting it's a function)
In AppStatusDisplay it's a noun